mbtools/validate_mbz_html.py

```diff
@@ -115,6 +117,9 @@ def run_validations(html_elements, include_styles):
     violations.extend(find_tag_violations(html_elements))
     violations.extend(find_nested_ib_violations(html_elements))
     violations.extend(find_ib_uuid_violations(html_elements))
+    if include_questionbank:
+        violations.extend([Violation(violation, 'questions.xml')for violation
```

Comment:
Three comments on this `Violation` object:

1. This is another pattern / consistency thing, but I think we need to be incorporating constants like the following into these violation issue parameter (some of this will be easier to do once you pull the validation logic into this file vs `models.py`):

```python
DUPLICATE_QBANK_UUID_VIOLATION = "ERROR: Duplicate question bank UUID"
INVALID_QBANK_UUID_VIOLATION = "ERROR: Invalid question bank UUID"
```

2. You'll also be able to use the location string provided by `question_bank.mbz_path` instead of hard coding things here.

3. Finally, let's use the `link` field to pass the UUID value too in both error types.